src/net/sourceforge/kolmafia/persistence/DailyLimitDatabase.java

```diff
@@ -112,7 +122,7 @@ public int getMax() {
         }
       }
 
-      return 1;
+      return -1;
```

Comment:
still no explanation of magic number here

Reply:
Wait what

Reply:
Forgot to push

Reply:
Please make note of the commit time for that commit! Haha